x-pack/test/observability_functional/apps/observability/pages/alerts/add_to_case.ts

```diff
@@ -12,7 +12,7 @@ export default ({ getService, getPageObjects }: FtrProviderContext) => {
   const observability = getService('observability');
   const retry = getService('retry');
 
-  describe('Observability alerts / Add to case', function () {
+  describe('Observability alerts / Add to case >', function () {
```

Comment:
Love this, makes it way easier to find the corresponding test in failing ftr logs